adafruit_il0373.py

```diff
@@ -69,7 +69,25 @@
 )
 # pylint: disable=too-few-public-methods
 class IL0373(displayio.EPaperDisplay):
-    """IL0373 driver"""
+    R"""IL0373 driver
```

Comment:
Is this R prefix intentional? CircuitPython can't parse it.

Reply:
Weird, CPython treats R and r the same.
I fixed this and pushed.

Reply:
What is the r prefix for? What version of Python was it added in? Did you verify CircuitPython can parse the lower case r?

Reply:
the r prefix is for a raw string literal, meaning that escape sequences are not evaluated.
This is mostly useful for filesystem paths which may contain escape sequences.
This has been around since at lease python 2.6.
circuitpython supports raw string literals with a lower case r but not an upper case r.
CPython supports both.
See attached image for use in circuitpython.

![image](https://user-images.githubusercontent.com/1206799/72296475-4d4fed00-3662-11ea-8478-c9e612763a07.png)